README.md

```diff
@@ -91,6 +91,12 @@ app.listen(80, function () {
 
 ### Configuring CORS w/ Dynamic Origin
 
+Provide an `origin` function, which will be passed a string `origin` containing the origin domain, and a `callback` expecting `(err, ok)`.
+
+If you callback with `(null, true)`, the incoming origin will be echoed back in the access-control-allow-origin header, permitting browser acces to the resource. Callback with `(null, false)` and it won't, and the browser will abort the cross-origin request.
```

Comment:
So the second argument can be more than `true`/`false`; it can be anything that the `origin` option can be, like a string or a regular expression, etc.